app/src/main/java/com/orange/ouds/app/ui/tokens/TokenPropertyIllustration.kt

```diff
@@ -142,7 +139,8 @@ fun SizeIconWithTextIllustrationRow(size: Dp, tokenName: String) {
         val style = OudsTypographyKeyToken::class.getTokens()
             .asOrNull<List<Token<TextStyle>>>()
             ?.firstOrNull { typographyToken ->
-                tokenName.split(".").firstOrNull() == typographyToken.name.replace("Strong", "").replace(".", "")
+                val tokenNameTreeValues = tokenName.split('.')
+                typographyToken.name.replace(".Strong", "") == "${tokenNameTreeValues[0]}.${tokenNameTreeValues[1]}"
```

Comment:
This can be slightly simplified:

`typographyToken.name.replace(".Strong", "") == tokenName.split('.').take(2).joinToString(".")`

And maybe we could even compute `tokenName.split('.').take(2).joinToString(".")` outsite of `firstOrNull` to avoid multiple processing.